workflows/comfyui/sd15-multi-cnet-depthanything-face.json

```
@@ -869,6 +869,47 @@
         2
       ]
     },
+    {
+      "id": 26,
+      "type": "LoadImage",
+      "pos": [
+        100,
+        1142
+      ],
+      "size": [
+        315,
+        314
+      ],
+      "flags": {},
+      "order": 5,
+      "mode": 0,
+      "inputs": [],
+      "outputs": [
+        {
+          "name": "IMAGE",
+          "type": "IMAGE",
+          "links": [
+            24,
+            58
+          ],
+          "slot_index": 0
+        },
+        {
+          "name": "MASK",
+          "type": "MASK",
+          "links": null
+        }
+      ],
+      "properties": {
+        "cnr_id": "comfy-core",
+        "ver": "0.3.18",
+        "Node name for S&R": "LoadImage"
+      },
+      "widgets_values": [
+        "harold.png",
```

Let's make harold more generic
```suggestion
        "example.png",
```